### PR TITLE
Gh 19 jobs page

### PR DESCRIPTION
**Pull request by joshmcintosh**
A page is needed where a user can view their currently running (and finished) jobs. This page must show the status of each job, for example "running" or "completed" Further, each user should have their own "jobs" page which is only accessible by them.

### Acceptance Criteria
 * A Jobs page has been created
 * The Jobs page has a list of the currently running jobs, with an indication saying that they are running
 * The Jobs page has a list of the finished jobs, with an indication saying that they are finished
 * The Jobs page displays only for the user that made those jobs.

### CODE REVIEW

**Comment by Ross-Alexandra** [comment]
After tests on local, it appears that all the AC are met, I'll review shortly.

File: ccf/app/views.py [hunk 1]
```diff
@@ -38,6 +39,18 @@ def job_create_view(request):
     return render(request, "job_create.html", context)
 
 
+@login_required
+def jobs_view(request):
+    """Jobs Page
+
+    Returns to the user all of their jobs and their status.
+    """
+    jobs_list_unfin = Job.objects.filter(user=request.user).filter(finished=False)
```
Comment (Ross-Alexandra) [inline ccf/app/views.py]: somewhat pedantic, but calling this `unfinished_jobs` and `finished_jobs` is the same number of characters, but considerably more readable.

File: ccf/tests/test_urls.py [hunk 2]
```diff
@@ -83,3 +83,22 @@ def test_post_job_when_not_logged_in_redirects(self):
         SimpleTestCase().assertRedirects(
             response, "/accounts/login/?next=/jobs/create/"
         )
+
```
Comment (Ross-Alexandra) [inline ccf/tests/test_urls.py]: using something along the lines of 
```
@patch("app.views.jobs_view.render")
def test_jobs_lists_are_correct(render_mock):
    # Test code here
    render_mock.assert_called_with(dict_with_correct_values)
```

We can probably test here that the lists being returned definitely contain only jobs for that user, and that all the correct jobs are in the correct places. Might not be worth the time with the due date as soon as it is, but definitely worth thinking about.

File: ccf/app/views.py (hunk 1 above)
Comment (joshmcintosh) [reply ccf/app/views.py]: Yeah, it's probably a good idea for me to change that, thanks.

**Comment by Ross-Alexandra** [comment]
Current working theory on the failed build is something going on with django (we don't list sqlite as a dependency) however this could be something to do with versioning. Looking further into this.

**Comment by joshmcintosh** [comment]
@Ross-Alexandra Ready for review, boss.

**Pull request merged**